tests/phpunit/Unit/Query/DescriptionFactoryTest.php

```diff
@@ -225,4 +225,35 @@ public function testCanConstructDescriptionFromValidDataValue() {
 		);
 	}
 
+	public function testCanConstructDescriptionFromMonolingualTextValue() {
+
+		$containerSemanticData = $this->getMockBuilder( '\SMWContainerSemanticData' )
+			->disableOriginalConstructor()
+			->getMock();
+
+		$containerSemanticData->expects( $this->atLeastOnce() )
+			->method( 'getPropertyValues' )
+			->will( $this->returnValue( array( $this->dataItemFactory->newDIBlob( 'Bar' ) ) ) );
+
+		$dataValue = $this->getMockBuilder( '\SMW\DataValues\MonolingualTextValue' )
```

Comment:
Because I know what I want to test and setting up a real object is a pain and doesn't help here as I want a specific condition.